src/BinDeps.jl

```diff
@@ -47,8 +47,8 @@ module BinDeps
     function download_cmd(url::AbstractString, filename::AbstractString)
         global downloadcmd
         if downloadcmd === nothing
-            for download_engine in @windows? ("C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell",
-                    :powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch)
+            for download_engine in (@static is_windows()? ("C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell",
```

Comment:
`@static` isn't necessary here (and the deprecation message should probably be changed to reflect this).